.travis.yml

```diff
@@ -1,9 +1,15 @@
 dist: trusty
+sudo: required
+before_install:
+  - source .travis/before_install_$TRAVIS_OS_NAME.sh
 git:
   # use depth 2 just in case two refs get pushed at once (like a tag)
   depth: 2
 language: ruby
-cache: bundler
+cache:
+  bundler: true
```

Comment:
I don't want to cache bundler because I want to verify installation works from scratch. We have had problems in the past an installation time and we need to verify it works.

Reply:
Rereading the manual at https://docs.travis-ci.com/user/caching/ makes me believe my change would have been a noop, as it inspects where the gems go with the --path and caches those.
cache: { bundler: true } is only required if there are other directories in need of caching, so let's leave it as is.

Reply:
I just don't trust the cache, so I don't want it to be anywhere in the CI config.